pages/api/query.js

```diff
@@ -15,9 +20,7 @@ const allProjections = `
     variants {
       ...,
       arrayOfTitleAndText[] {
```

Comment:
For this part of the query, kindly update to `arrayOfTitleAndText[]` if we're just going to get data from array as is.

Reply:
this query just return the title and text in array format.